docs/3.0.0-beta.x/guides/email.md

```diff
@@ -28,15 +28,15 @@ You can check all the available providers developed by the community on npmjs.or
 To install a new provider run:
 
 ```
-$ npm install strapi-provider-email-sendgrid@alpha --save
+$ npm install strapi-provider-email-sendgrid@beta --save
 ```
 
 ::: note
-If the provider is not in the mono repo, you probably not need `@alpha` depending if the creator published it with this tag or not.
+If the provider is not in the mono repo, you probably don't need `@beta` depending if the creator published it with this tag or not.
```

Comment:
This sentence is unclear to me. 

Maybe this is more clear:

Installing a new provider that is in the mono repo requires using the @beta tag. Providers not contained in the mono repo will have documentation stating whether this tag is needed or not.